{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-02-21",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
      "Discloser": "B3EF",
      "Fixer": ""
    },
    "Package": {
      "Registry": "pip",
      "Name": "altair",
      "URL": "https://pypi.org/project/altair",
      "Downloads": ""
    },
    "CWEs": [
      {
        "ID": "94",
        "Description": "Arbitrary Code Execution"
      }
    ],
    "CVSS": {
        "Version": "3",
        "AV": "N",
        "AC": "L",
        "PR": "L",
        "UI": "N",
        "S": "U",
        "C": "H",
        "I": "H",
        "A": "H",
        "E": "X",
        "RL": "X",
        "RC": "X",
        "Score": "8.8"
    },
    "CVEs": [""],
    "Repository": {
      "URL": "https://github.com/altair-viz/altair",
      "Codebase": ["Python"],
      "Owner": "Altair",
      "Name": "altair",
      "Forks": "578",
      "Stars": "6.4K"
    },
    "Permalinks": ["https://github.com/altair-viz/altair/blob/master/altair/_magics.py#L123"],
    "References": [
      {
        "Description": "",
        "URL": ""
      }
    ],
    "PrNumber": "1942"
  }
